conf/layer.conf

```diff
@@ -3,12 +3,19 @@ BBPATH .= ":${LAYERDIR}"
 
 # We have recipes-* directories, add to BBFILES
 BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
-            ${LAYERDIR}/recipes-*/*/*.bbappend \
             ${LAYERDIR}/images/*.bb"
 
+BBFILES += " \
+./recipes-multimedia/gstreamer/gstreamer1.0-plugins-ugly_%.bbappend \
+./recipes-connectivity/openssh/openssh_8%.bbappend\
+"
+
+BBFILES_append_odroid-n2 = " ./recipes-kernel/linux/linux-yocto_5.10.bbappend"
+BBFILES_append_raspberrypi4-64 = " ./recipes-bsp/bootfiles/rpi-config_git.bbappend"
```

Comment:
Here as well.

Reply:
Took a different approach here, `bitbake-layers show-appends` output looks ok, testing image in progress.

Reply:
Great, works on my side.